x-pack/plugins/security_solution/public/management/cypress/e2e/serverless/metering.cy.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { recurse } from 'cypress-recurse';
+import type { UsageRecord } from '@kbn/security-solution-serverless/server/types';
+import {
+  getInterceptedRequestsFromTransparentApiProxy,
+  startTransparentApiProxy,
+  stopTransparentApiProxy,
+} from '../../tasks/transparent_api_proxy';
+import type { ReturnTypeFromChainable } from '../../types';
+import { indexEndpointHeartbeats } from '../../tasks/index_endpoint_heartbeats';
+import { login, ROLE } from '../../tasks/login';
+
+describe(
+  'Metering',
+  {
+    tags: ['@serverless', '@skipInServerlessMKI'],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolutionServerless.usageReportingTaskInterval=1m`,
+          `--xpack.securitySolutionServerless.usageReportingApiUrl=https://localhost:3623`,
+        ],
+      },
+    },
+  },
+  () => {
+    const HEARTBEAT_COUNT = 2001;
```

Comment:
not a big deal but would be good to use the actual `BATCH_SIZE` variable here and below in case we end up changing it one day.

Reply:
Exported METERING_SERVICE_BATCH_SIZE from `security-solution-serverless/server/constants`, got rid of hardcoded `1000`'s